src/kata6.joinNames.js

```diff
@@ -1,3 +1,13 @@
-const joinNames = (namesObj) => {};
+const joinNames = (namesObj) => {
+  let string = "";
+  for (let i = 0; i < namesObj.length; i++) {
+    if (i === namesObj.length - 2) {
+      string += `${namesObj[i].name} & ${namesObj[i + 1].name}`;
+      break;
+    }
+    string += `${namesObj[i].name}, `;
+  }
+  return string;
+};
```

Comment:
the simplest way to fix your code for if the namesObj has a length of 1 is to add the line `if (namesObj.length === 1) return namesObj[0].name;` at the top of the function, before `let string...`